recipes/bz2file/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "bz2file" %}
+{% set version = "0.98" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 64c1f811e31556ba9931953c8ec7b397488726c63e09a4c67004f43bdd28da88
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - bz2file
+
+about:
+  home: https://github.com/nvawda/bz2file
+  license: Apache 2.0
+  #####################################################
+  # No license file. :/                               #
+  #                                                   #
+  # xref: https://github.com/nvawda/bz2file/issues/5  #
+  #####################################################
+  # license_file: LICENSE.txt
```

Comment:
Unfortunately there is no license file. Not even on GitHub. Have raised issue ( https://github.com/nvawda/bz2file/issues/5 ).